himl/simplevault.py

```diff
@@ -23,12 +24,18 @@ def __init__(self):
     def get_vault_client(self):
         url = os.getenv('VAULT_ADDR')
         namespace = os.getenv('VAULT_NAMESPACE')
+        verify = not strtobool(os.getenv('VAULT_SKIP_VERIFY', 'false'))
+        
+        if not verify:
+            import urllib3
+            urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
It should be clear for the user if the connection is insecure. We should not suppress warnings.

Reply:
I just re-enabled the warning.